3Dmol/glviewer.js

```diff
@@ -3220,14 +3218,19 @@ $3Dmol.GLViewer = (function() {
             }
             var mostFrames = this.getNumFrames();
             var that = this;
-            var currFrame = 0;
+
+            var startFrame = typeof options.startFrame === 'number' ? options.startFrame : 0;
+            var currFrame = startFrame;
+
+            var numFramesToAnimate = loop === "backward" ? startFrame : mostFrames - startFrame;
```

Comment:
I don't think this is right for "backAndForth" - in that case it should cycle through all the frames ending back at the start frame, so numFramesToAnimate should be mostFrames